src/components/contents/ListView.jsx

```diff
@@ -15,9 +15,19 @@ function ListView({ user, currentDBId }) {
 
   const { data, isLoading } = useQuery(["dbDocumentList"], getDocumentsList, {
     enabled: !!user,
+    onSuccess: result => {
+      const newArr = [];
+
+      result.data.database.documents.forEach(element => {
+        return newArr.push(element._id);
+      });
+
+      setDocumentsIds(newArr);
```

Comment:
```suggestion
      const newArr = result.data.database.documents.map(element => {
        return element._id;
      });

      setDocumentsIds(newArr);
```

forEach는 애초에 return 값이 없기 때문에 굳이 return을 안해주셔도 괜찮을것 같습니다! 
만약에 리턴 값이 필요하다면 해당 함수를 forEach가 아닌 map으로 사용해보는건 어떠신가요?